cmd/juno/juno.go

```diff
@@ -148,7 +149,7 @@ const (
 		"These peers can be either Feeder or regular nodes."
 	p2pFeederNodeUsage = "EXPERIMENTAL: Run juno as a feeder node which will only sync from feeder gateway and gossip the new" +
 		" blocks to the network."
-	p2pPrivateKeyUsage   = "EXPERIMENTAL: Hexadecimal representation of a private key on the Ed25519 elliptic curve."
+	p2pPrivateKeyUsage   = "EXPERIMENTAL: Hexadecimal representation of a private key on the Ed25519 elliptic curve. Also accepts a file path."
```

Comment:
I am unsure whether it is a good idea to have a flag accept 2 different input types.

Reply:
I see. My thought process is having one flag can abstract out the input process.

That could be confusing. Perhaps it's better to have a separate flag to accept file path for private key. Let me work on that.